R/mainTests.r

```diff
@@ -1001,34 +1002,44 @@ overcompleted_cols <- function(data, meta) {
     # This is a test that could benefit from more detail, and maybe a table in the error feedback
 
     # Start by filtering the data down to remove the geographic level being tested, lad rows and any lower levels we don't care about
+    levels_compatible_with_la <- c("Local authority", "Local authority district", "School", "Provider", "Institution", "Planning area")
 
     level_rows <- data %>%
-      filter(geographic_level != matrixRow[1]) %>%
-      filter(!geographic_level %in% geography_matrix[13:16, ]) %>%
-      filter(geographic_level != geography_matrix[4, 1])
+      filter(!geographic_level %in% levels_compatible_with_la)
```

Comment:
Same question here as above, should we be filtering out the level that is being tested as well (matrixRow)?

Reply:
Same as above, it's being filtered out using that one filter line as it's included in the levels_compatible_with_la vector.